apps/passport-server/test/devconnect.spec.ts

```diff
@@ -635,6 +642,87 @@ describe("devconnect functionality", function () {
     );
   });
 
+    /**
+   * This covers the case where we have a ticket marked as consumed, but
+   * the check-in is cancelled in Pretix.
```

Comment:
'check-in is cancelled' is kind of confusing terminology - do you mean that it is 'effectively cancelled out by a check-in exit entry' or that a 'check-in entry was cancelled' or that a 'check-in entry was deleted altogether'?